tracer.go

```diff
@@ -116,7 +114,7 @@ type Tracer interface {
 
 // StartSpanOptions allows Tracer.StartSpan() callers and implementors a
 // mechanism to override the start timestamp, specify Span References, and make
-// Tags available at Span start time.
+// a single Tag or multiple Tags available at Span start time.
```

Comment:
A quick prose fix would be something along the lines of `and make instances of Tag or a Tags available at Span start time`. I'm not 100% happy with that one, either, though.